main.py

```diff
@@ -136,7 +136,7 @@ def validate_data_folder(app_data_folder: Path) -> tuple:
         return (app_data_folder / 'secrets.yaml', app_data_folder / 'config.yaml', app_data_folder / 'plain_text_resume.yaml', output_folder)
 
     @staticmethod
-    def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path) -> dict:
+    def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path, cover_letter_file: Path | None = None) -> dict:
```

Comment:
Boy Scout Code Principle

Can you move FileManager to a different file maybe file_manager.py

rename method file_paths_to_dict to something meaning full. refactor method if possible